Resources/public/js/adapter/fancytree.js

```diff
@@ -13,8 +13,14 @@ var FancytreeAdapter = function (options) {
         throw 'The FancytreeAdapter requires a request option.';
     }
 
+    var alwaysTrueFunction = function (node, data) { return true; };
     this.requestData = options.request;
     this.rootNode = options.root_node || '/';
+    this.dndOptions = jQuery.extend({
+        enabled: false,
+        isNodeDraggable: alwaysTrueFunction,
+        nodeAcceptsDraggable: alwaysTrueFunction
```

Comment:
this one specifies if a node accepts the dragged node to be added as child (e.g. based on the valid_children config)